### DISCUSSION TITLE
Allure Karate integration support

### DISCUSSION THREAD

**Discussion by vivekcalls** [allure-framework/allure2#1990] [General Discussion]
Hi,

I'm trying to integrate Allure reporting for Karate. I have added the allure-karate dependency.

When I run my test I see only the result.json file and on opening the report using allure serve I see blank report with message "allure-results does not exists"

Please let me know what is that I'm missing

Thanks
Vivek

**Comment by baev** [comment]
There is an example available https://github.com/allure-examples/allure-karate-example
